test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -1581,12 +1571,8 @@ updateStats action lookUp modelBefore _modelAfter result =
 
 -- | Tags for every step
 data Tag =
-    -- | (At least) two types of tables were created (i.e., 'New') in the same
-    -- session. The strings represent the representations of the types that the
-    -- tables were created at.
-    NewTwoTableTypes String String
```

Comment:
This comment got lost somehow, so here goes again:

One could argue that we can keep these tags, because they are still achievable through writing our own `DL` examples, even if the generator does generate actions anymore that can trigger these tags

Reply:
I see, you do mean DL examples. Fair enough.